frontend/lib/hp-action.tsx

```diff
@@ -24,39 +24,81 @@ import { HarassmentCaseHistory } from './pages/hp-action-case-history';
 import { BigList } from './big-list';
 import { EmailAttachmentForm } from './email-attachment';
 import { EmailHpActionPdfMutation } from './queries/EmailHpActionPdfMutation';
-import { CustomerSupportLink } from './customer-support-link';
 import { GetStartedButton } from './get-started-button';
 import { AccessForInspection } from './pages/hp-action-access-for-inspection';
 import { HPActionYourLandlord } from './pages/hp-action-your-landlord';
 import { GeneratePDFForm, ShowHPUploadStatus } from './pages/hp-action-generate-pdf';
 import { isNotSuingForRepairs, isNotSuingForHarassment, hasFeeWaiverAnd } from './hp-action-util';
+import { StaticImage } from './static-image';
+import { MoratoriumWarning } from './covid-banners';
+
+const HP_ICON = "frontend/img/hp-action.svg";
 
 const onboardingForHPActionRoute = () => getSignupIntentOnboardingInfo(OnboardingInfoSignupIntent.HP).onboarding.latestStep;
 
+// HP Cases currently being accepted in Housing Court amidst COVID-19 crisis
+const acceptedCases = [
+  "no heat", 
+  "no hot water", 
+  "no gas", 
+  "mold", 
+  "lead-based paint", 
+  "no working toilet", 
+  "vacate order issued"
+];
```

Comment:
Oh uhh there are ways we can avoid this to keep things DRY.  will file an issue to deal w/ it later.

Reply:
ok gotcha— yeah suspected as such